### PR TITLE
fix(frontend): added dynamic color on logbook timestamp

### PR DESCRIPTION
**Pull request by Millefeuille42**
#6469 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by floreal15 — COMMENTED** [review]
review done and the code is fine

File: frontend_app_logbook/src/css/index.styl [hunk 1]
```diff
@@ -148,7 +152,7 @@ cardWidth = 280px
             background-color lightGrey2
         &:empty
           display none
-
+w
```
Comment (Skylsmoi) [inline frontend_app_logbook/src/css/index.styl:155]: typo

**Review by Skylsmoi — APPROVED** [review]

File: frontend_app_logbook/src/css/index.styl (hunk 1 above)
Comment (Millefeuille42) [reply frontend_app_logbook/src/css/index.styl]: fixed

**Review by Skylsmoi — APPROVED** [review]

File: frontend_app_logbook/src/css/index.styl [hunk 2]
```diff
@@ -90,6 +89,11 @@ cardWidth = 280px
             justify-content end
             &__edit
               margin-inline-end standardSpacing
+  &__alt
+    &__lightTextColor
+      color lightGrey1
+    &__darkTextColor
+      color darkGrey2
```
Comment (Skylsmoi) [inline frontend_app_logbook/src/css/index.styl:96]:
```suggestion
  & .lightTextColor
    color lightGrey1
  & .darkTextColor
    color darkGrey2
```

**Review by Skylsmoi — APPROVED** [review]

**Comment by Skylsmoi** [comment]
tested on branch

**Pull request merged**